Add dynamo.yml and update iam.yml

dannydaviesIW opened the pull request:
# Description

-   This PR adds a new cloudformation template for creating a dynamodb table to be used with sessions on the site.
-   The existing iam.yml cloudformation template has also been updated to include extra permissions for the site to read from/write to dynamodb.

Related PRs:
-   `fdbt-site` repo - https://github.com/fares-data-build-tool/fdbt-site/pull/109
-   `fdbt-dev` repo - https://github.com/fares-data-build-tool/fdbt-dev/pull/13

dannydaviesIW committed:
Added dynamo.yml

dannydaviesIW committed:
Updated table name

dannydaviesIW committed:
Added dynamodb permissions to iam.yml

ljones92 commented on cloudformation/service/eu-west-2/dynamo.yml:
```diff
@@ -0,0 +1,34 @@
+        - AttributeName: !Ref "HashKeyName"
+          KeyType: HASH
+      TimeToLiveSpecification:
+        AttributeName: ttl
```
This will need to be `expires` for the default settings in the session library

ljones92 commented on cloudformation/service/eu-west-2/iam.yml:
```diff
@@ -122,6 +122,12 @@ Resources:
                   - "cognito-idp:DescribeUserPoolClient"
                   - "cognito-idp:ListUsers"
                 Resource: !Sub arn:aws:cognito-idp:${AWS::Region}:${AWS::AccountId}:userpool/*
+              - Effect: Allow
```
We'll need a `Resource: "*"` on here as well

dannydaviesIW committed:
Updated dynamo.yml, iam.yml and fargate.yml cloudformation templates

- Added SESSION_SECRET env var as a parameter in fargate.yml
- Added resource attribute to dynamo permissions in iam.yml
- Amended TimeToLiveSpecification in dynamo/yml

dannydaviesIW committed:
Merge branch 'master' into feature/TFN-549_setup_express_sessions
